src/java/org/apache/cassandra/io/sstable/format/trieindex/TrieIndexFormat.java

```diff
@@ -348,7 +348,7 @@ static class TrieIndexVersion extends Version
             hasOriginatingHostId = version.matches("(a[d-z])|(b[b-z])") || version.compareTo("ca") >= 0;
             hasMaxColumnValueLengths = version.matches("b[a-z]"); // DSE only field
             correspondingMessagingVersion = version.compareTo("ca") >= 0 ? MessagingService.VERSION_SG_10 : MessagingService.VERSION_3014;
-            hasExplicitlyFrozenTuples = version.compareTo("cc") < 0 || version.compareTo("da") >= 0; // we don't know if what DA is going to be eventually, but it is almost certain it will not include explicitly frozen tuples
+            hasImplicitlyFrozenTuples = version.compareTo("cc") < 0 || version.compareTo("da") >= 0; // we don't know if what DA is going to be eventually, but it is almost certain it will not include implicitly frozen tuples
```

Comment:
also the comment is probably off, hmm...

Reply:
that is - "DA ... will include implicitly frozen tuples" (because the patch was not applied on OSS... yet)

Reply:
will adjust